games-util/springlobby/springlobby-0.264.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0"
+inherit cmake-utils wxwidgets
+
+DESCRIPTION="The official lobby client for SpringRTS community games"
+HOMEPAGE="https://springlobby.info"
+SRC_URI="https://www.springlobby.info/tarballs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
```

Comment:
Why?

Reply:
Why what?
LICENSE is GPLv2+
restrict is/was there as there were no distfiles on mirrors.

Reply:
I meant RESTRICT. You need to restrict mirroring only if it's forbidden for some reason. Mirrors will fetch it once it's pushed.

As for the license, make it `GPL-2+` then (there's such file in Gentoo).

Reply:
Done